src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html

```diff
@@ -83,28 +84,19 @@
               (click)="globalVars.hasUserBlockedCreator(profile.PublicKeyBase58Check) ? unblock() : block()"
               *ngIf="!profileBelongsToLoggedInUser()"
             >
-              {{ globalVars.hasUserBlockedCreator(profile.PublicKeyBase58Check) ? 'creator_profile_top_card.unblock' : 'creator_profile_top_card.block' | transloco }} {{ 'creator_profile_top_card.user' | transloco }}
+              {{
+                globalVars.hasUserBlockedCreator(profile.PublicKeyBase58Check)
+                  ? "creator_profile_top_card.unblock"
+                  : ("creator_profile_top_card.block" | transloco)
+              }}
+              {{ "creator_profile_top_card.user" | transloco }}
             </a>
             <a
               class="dropdown-menu-item d-block link--unstyled p-10px feed-post__dropdown-menu-item"
               (click)="reportUser()"
               *ngIf="!profileBelongsToLoggedInUser()"
             >
-              {{ 'creator_profile_top_card.report_user' | transloco }}
-            </a>
-            <a
-              class="dropdown-menu-item d-block link--unstyled p-10px feed-post__dropdown-menu-item"
-              (click)="updateWellKnownCreatorsList()"
-              *ngIf="!profile.IsFeaturedTutorialUpAndComingCreator && globalVars.showSuperAdminTools()"
-            >
-              {{ profile.IsFeaturedTutorialWellKnownCreator ? 'creator_profile_top_card.remove' : 'creator_profile_top_card.add' | transloco }} {{ 'creator_profile_top_card.user' | transloco }} {{ 'creator_profile_top_card.wellknown_creator' | transloco }}
-            </a>
-            <a
-              class="dropdown-menu-item d-block link--unstyled p-10px feed-post__dropdown-menu-item"
-              (click)="updateUpAndComingCreatorsList()"
-              *ngIf="!profile.IsFeaturedTutorialWellKnownCreator && globalVars.showSuperAdminTools()"
-            >
-              {{ profile.IsFeaturedTutorialUpAndComingCreator ? 'creator_profile_top_card.remove' : 'creator_profile_top_card.add' | transloco }} {{ 'creator_profile_top_card.user' | transloco }} {{ 'creator_profile_top_card.upcoming_creator' | transloco }}
```

Comment:
to check

Reply:
These are a part of the tutorial, and are no longer used